azext_edge/edge/providers/orchestration/targets.py

```diff
@@ -150,7 +153,11 @@ def get_ops_enablement_template(
             },
             template_blueprint=M3_ENABLEMENT_TEMPLATE,
         )
-
+        if self.user_trust:
+            # disable cert and trust manager
+            parameters["trustConfig"]["value"]["source"] = "CustomerManaged"
+            # patch enablement template expecting full trust settings for source: CustomerManaged
+            del template.content["definitions"]["_1.CustomerManaged"]["properties"]["settings"]
         # TODO - @digimaun - expand trustSource for self managed & trustBundleSettings
```

Comment:
This todo can be deleted, it was for m2 which was completed and forgot to remove it